api/discovery_chain_test.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// testClusterID is the Consul cluster ID for testing.
```

Comment:
ubet nit: Copy paste is fine but is it worth commenting that this is duplicated in `connect` package just in case we ever choose to change it?

Reply:
Added note